canvases/dark-editor-vue/src/main.ts

```diff
@@ -2,13 +2,25 @@ import { createApp } from 'vue'
 import App from './App.vue'
 import './assets/tailwind.css'
 
+/* import the fontawesome core */
+import { library } from '@fortawesome/fontawesome-svg-core'
+
+/* import font awesome icon component */
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+/* import specific icons */
+import { fas } from '@fortawesome/free-solid-svg-icons'
```

Comment:
Here should I import only the solid icons that we are using instead of importing all of them?
Reminder: if the answer is no, update this comment /* import specific icons */

Reply:
Hmm. I'm unsure. Maybe let's ask in Discord, and consider this as a follow-up PR? I plan to merge this PR shortly